src/utils/transform/manifest.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Check if a function file name matches an entry in the manifest.
+ *
+ * @param entry Manifest entry name.
+ * @param file Function file name.
+ * @returns Whether the function file name matches the manifest entry name.
+ */
+export const matchFunctionEntry = (entry: string, file: string) => {
```

Comment:
nit, maybe it would be clearer to include name in the params names?
```suggestion
export const matchFunctionEntry = (entryName: string, fileName: string) => {
```